Source/Core/VideoCommon/DriverDetails.h

```diff
@@ -74,7 +74,7 @@ namespace DriverDetails
 		// Pinned memory is disabled for index buffer as the AMD driver (the only one with pinned memory support) seems
 		// to be broken. We just get flickering/black rendering when using pinned memory here -- degasus - 2013/08/20
 		// This bug only happens when paired with base_vertex.
-		// Please see issue #6105 on Google Code. Let's hope buffer storage solves this issues.
+		// Please see issue #6105. Let's hope buffer storage solves this issue.
```

Comment:
Yes I noticed that too but I assume since the code is still in use that the comment should be left.